ci/before_deploy.sh

```diff
@@ -7,6 +7,9 @@ package() {
     fi
 
     cp -f target/$TARGET/release/$BIN_NAME bin/$BIN_NAME_TAG
+    cd bin || return 1
+    sha256sum "$BIN_NAME_TAG" | tee "$BIN_NAME_TAG.sha256"
```

Comment:
Thanks for the work!

Can you update this a little bit to
```
sha256sum "$BIN_NAME_TAG" | tee "bin/$BIN_NAME_TAG.sha256"
```
rather than changing dirs twice?

Reply:
Thanks sure done in 6d6d7e0947911a4e5c25a06d6eb0133a96a56bd1